src/sentry/processing/backpressure/rabbitmq.py

```diff
@@ -53,40 +31,18 @@ def _parse_rabbitmq(host: Dict[str, str]) -> RabbitMqHost:
     if password is None:
         raise ValueError("missing password")
 
-    return RabbitMqHost(hostname, port, vhost, username, password)
-
-
-def _prefix_key(key_name: str) -> str:
-    return f"bp1:{key_name}"
-
-
-def _unhealthy_consumer_key(consumer_name: str) -> str:
-    return _prefix_key(f"{UNHEALTHY_KEY_NAME}:{consumer_name}")
-
+    if vhost is None:
+        raise ValueError("missing vhost")
 
-def is_consumer_healthy(consumer_name: str = "default") -> bool:
-    """Checks whether the given consumer is healthy by looking it up in Redis.
+    return RabbitMqHost(hostname, port, vhost, username, password)
 
-    NB: If the consumer is not found in Redis, it is assumed to be healthy.
-    This behavior might change in the future.
-    """
 
-    if not options.get("backpressure.monitor_queues.enable_check"):
-        return True
-    # check if queue is healthy by pinging Redis
-    try:
-        # We set the key if the queue is unhealthy. If the key exists,
-        # the queue is unhealthy and we need to return False.
-        healthy = not queue_monitoring_cluster.exists(_unhealthy_consumer_key(consumer_name))
-        # TODO: do we want to also check the `default` consumer as a catch-all?
-    except Exception as e:
-        sentry_sdk.capture_exception(e)
-        # By default it's considered healthy
-        healthy = True
-    return healthy
+def query_rabbitmq_memory_usage(host: RabbitMqHost) -> Tuple[int, int]:
+    # TODO: actually get the memory usage :-)
+    return (0, 1)
```

Comment:
Comment for visibility. This will be the hook point for https://github.com/getsentry/team-processing/issues/55 and its implementation via https://github.com/getsentry/sentry/pull/50631